CMSgov/mint-app
# [MINT-2179, MINT-2987] Update PBG: Help Text & Questions

# MINT-2179, [MINT-2987](https://jiraent.cms.gov/browse/MINT-2987)

## Description

- Added new help text in the following sections
  - General Characteristics section
    - Is care coordination involved?
    - Are community partners involved?
    - Which types of waivesr are required? Select all that apply. 
- Added new questions in Payment section and Participants and providers section


## How to test this change

- Go through the edit view
- Check the General Characteristics section to verify help text is there
- Go to Payment section Page 1 --> add `Claims-Based Payments` Nav to Page 3 --> see new question
- Go to Participants and providers page 1 -> add `Medicare providers/supplies` and see new question
- Check data persists
- Go to Read Only views
- Test the CSV export

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
